convert2rhel/unit_tests/subscription_test.py

```diff
@@ -144,6 +144,46 @@ def __call__(self, *args, **kwargs):
         return [namedtuple("Sub", ["pool_id", "sub_raw"])("samplepool", "Subscription description")]
 
 
+class PromptUserMocked(unit_tests.MockFunction):
+    def __call__(self, *args, **kwargs):
+        return True
+
+
+class TestCheckNeededReposAvailability(object):
+    def test_check_needed_repos_availability(self, monkeypatch, caplog):
+        monkeypatch.setattr(utils, "ask_to_continue", PromptUserMocked())
+        monkeypatch.setattr(subscription, "get_avail_repos", lambda: ["rhel_x", "rhel_y"])
+
+        avail_repos_message = "Needed RHEL repositories are available."
+        subscription.check_needed_repos_availability(["rhel_x"])
+
+        assert avail_repos_message in caplog.records[-1].message
+
+        no_avail_repos_message = (
+            "Some repositories are not available: rhel_z. "
+            " Some pacakges may not be replaced with their corrispomding"
+            " RHEL packages when converting. The converted system will be"
+            " running a mixture of packages form RHEL and from your current distibution"
```

Comment:
```suggestion
            "Some repositories are not available: rhel_z. "
            " Some pacakges may not be replaced with their corresponding"
            " RHEL packages when converting. The converted system will end up"
            " with a mixture of packages from RHEL and your current distibution." 
```

Reply:
I have completed this fix for the typo. Thank you for spotting this.